docs/modules/operation/pages/provisioning/snmp-encryption.adoc

```diff
@@ -0,0 +1,43 @@
+== Encrypting SNMP Credentials
+
+You can save SNMP credentials like readCommunity/WriteCommunity, authPassPhrase/privPassPhrase on the file system in encrypted format.
+
+=== Encryption methodology
+
+A random password is generated (once) for `snmp-config` alias and it is stored in java keystore file at `$OPENNMS_HOME/etc/scv.jce`.
+This password is used to encrypt/decrypt SNMP credentials.
+
+=== Enabling Encrption on SNMP Config
+
+Enable encryption on SNMP config by setting the following system property:
+[source, sh]
+----
+echo 'org.opennms.snmp.encryption.enabled=true' >> "$OPENNMS_HOME/etc/opennms.properties.d/snmp-encryption.properties"
+----
+
+SNMP config that is modified through the user interface will be encrypted directly on the file system.
```

Comment:
```suggestion
SNMP configuration modified through the user interface is encrypted directly on the file system.
```
I'm not clear if this is a comment about how the UI functions, or this is describing a result of what happens with anything that has been modified via the UI as a result of running this command.

Reply:
we are saying SNMP configuration that is modified on UI will get encrypted immediately on the file system.

Reply:
> > we are saying SNMP configuration that is modified on UI will get encrypted immediately on the file system.
> 
> Thanks. I updated my edit slightly.

Handled.